Gemfile

```diff
@@ -1,7 +1,8 @@
 source "https://rubygems.org"
 
 gem "rails", ">= 4.0.0"
-gem "kaminari"
+# FIXME: Fix garage to work with kaminari v1.
+gem "kaminari", "< 1"
```

Comment:
Oops, that's true.
https://github.com/cookpad/garage/blob/8ef010e538f24db6aed1b632acc03fa157a459dc/lib/garage/paginating_responder.rb#L56

But actually it's not required for now. We can make it optional if you don't paginate resources.